src/js/_enqueues/admin/site-icon.js

```diff
@@ -41,98 +66,178 @@
+	 *
+	 * @since 6.5.0
+	 */
+	$chooseButton.on( 'click', function() {
+		var $el = $( this );
+
+		// Create the media frame.
+		frame = wp.media({
+			button: {
+
+				// Set the text of the button.
+				text: $el.data( 'update' ),
+
+				// Don't close, we might need to crop.
+				close: false
+			},
+			states: [
+				new wp.media.controller.Library({
+					title: $el.data( 'choose-text' ),
+					library: wp.media.query({ type: 'image' }),
+					date: false,
+					suggestedWidth: $el.data( 'size' ),
+					suggestedHeight: $el.data( 'size' )
+				}),
+				new wp.media.controller.SiteIconCropper({
+					control: {
+						params: {
+							width: $el.data( 'size' ),
+							height: $el.data( 'size' )
+						}
+					},
+					imgSelectOptions: calculateImageSelectOptions
+				})
+			]
+		});
 
-			frame.on( 'cropped', function( attachment) {
-				$( '#site_icon_hidden_field' ).val(attachment.id);
-				switchToUpdate(attachment.url);
-				frame.close();
-				// Start over with a frame that is so fresh and so clean clean.
-				frame = null;
-			});
+		frame.on( 'cropped', function( attachment ) {
+			$hiddenDataField.val( attachment.id );
+			switchToUpdate( attachment );
+			frame.close();
+			// Start over with a frame that is so fresh and so clean clean.
+			frame = null;
+		});
 
-			// When an image is selected, run a callback.
-			frame.on( 'select', function() {
-				// Grab the selected attachment.
-				var attachment = frame.state().get('selection').first();
-
-				if ( attachment.attributes.height === $el.data('size') && $el.data('size') === attachment.attributes.width ) {
-					// Set the value of the hidden input to the attachment id.
-					$( '#site_icon_hidden_field').val(attachment.id);
-					switchToUpdate(attachment.attributes.url);
-					frame.close();
-				} else {
-					frame.setState( 'cropper' );
-				}
-			});
+		// When an image is selected, run a callback.
+		frame.on( 'select', function() {
+
+			// Grab the selected attachment.
+			var attachment = frame.state().get( 'selection' ).first();
+
+			if (
+				attachment.attributes.height === $el.data( 'size' ) &&
+				$el.data( 'size' ) === attachment.attributes.width
+			) {
 
-			frame.open();
+				// Set the value of the hidden input to the attachment id.
+				$hiddenDataField.val( attachment.id );
+				switchToUpdate( attachment.attributes );
+				frame.close();
+			} else {
+				frame.setState( 'cropper' );
+			}
 		});
+
+		frame.open();
 	});
 
-	function switchToUpdate( url ){
-		// Set site-icon-img src to the url and remove the hidden class.
-		$( '#site-icon-preview').find('img').not('.browser-preview').each( function(i, img ){
-			$(img).attr('src', url );
+	/**
+	 * Updates the UI when a site icon is selected.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @param {array} attributes The attributes for the attachment.
+	 */
+	function switchToUpdate( attributes ) {
+		var i18nAltString,
+		i18nAppAlternativeString,
+		i18nBrowserAlternativeString;
```

Comment:
```suggestion
		var i18nAltString, i18nAppAlternativeString, i18nBrowserAlternativeString;
```

Declerations can be grouped. 

https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#declaring-variables-with-var